client/planning-extension/src/assignments-overview/index.tsx

```diff
@@ -0,0 +1,245 @@
+            .then((currentUser) =>
+                Promise.all([
+                    fetchDesks(),
+                    fetchContentTypes(),
+                    fetchAssignments(currentUser._id),
+                ]).then(([desks, contentTypes, assignments]) => {
+                    this.setState({
+                        loading: false,
+                        currentUser: currentUser,
+                        desks: desks,
+                        contentTypes: contentTypes,
+                        assignments: assignments,
+                    });
+                })
+            );
+    }
+
+    componentWillUnmount() {
+        this.eventListenersToRemoveBeforeUnmounting.forEach((removeListener) => {
+            removeListener();
+        });
+    }
+
+    render() {
+        if (this.state.loading === true) {
+            return null;
+        }
+
+        const {assignments, desks, contentTypes} = this.state;
+        const itemsCount = assignments.length;
+        const grouped = groupBy(assignments, (item) => item.assigned_to.desk);
+
+        return (
+            <DropdownTree
+                groups={Object.keys(grouped).map((deskId) => ({
+                    render: () => (
+                        <GroupLabel>
+                            <Badge type="highlight" text={grouped[deskId].length.toString()} />
+                            <span style={{marginLeft: 6}}>{desks[deskId].name}</span>
+                        </GroupLabel>
+                    ),
+                    items: grouped[deskId],
+                }))}
+                getToggleElement={
+                    (isOpen, onClick) => (
+                        <TopMenuDropdownButton
+                            onClick={() => {
+                                if (itemsCount > 0) {
+                                    onClick();
+                                }
+                            }}
+                            active={isOpen}
+                            disabled={itemsCount < 1}
+                            pulsate={false}
+                            data-test-id="toggle-button"
+                        >
+                            <Badge type="highlight" text={itemsCount.toString()}>
+                                <span style={{color: isOpen ? '#3783A2' : 'white'}}>
+                                    <IconBig name="tasks" />
+                                </span>
+                            </Badge>
+                        </TopMenuDropdownButton>
+                    )
+                }
+                renderItem={(key, assignment, closeDropdown) => (
+                    <AssignmentsOverviewListItem
+                        key={key}
+                        assignment={assignment}
+                        contentTypes={contentTypes}
+                        onClick={closeDropdown}
+                    />
+                )}
+                wrapperStyles={{whiteSpace: 'nowrap', padding: 15, paddingTop: 0}}
```

Comment:
To stay in line with the UI framework's Layout Principles, padding should be `16` here